### PR TITLE
Add Material Black Theme

### PR DESCRIPTION
**Pull request by Maxmystere**
Feature for #317

Tested on Oneplus 8 with android 11

### CODE REVIEW

File: ultrasonic/src/main/java/org/moire/ultrasonic/util/Util.java [hunk 1]
```diff
@@ -153,7 +153,25 @@ public static boolean isLockScreenEnabled(Context context)
...
+		}
+		else if (Constants.PREFERENCES_KEY_THEME_BLACK.equalsIgnoreCase(theme))
+		{
+			context.setTheme(R.style.UltrasonicTheme_Black);
+		}
+		else if (Constants.PREFERENCES_KEY_THEME_LIGHT.equalsIgnoreCase(theme) || "fullscreenlight".equalsIgnoreCase(theme))
+		{
+			context.setTheme(R.style.UltrasonicTheme_Light);
+		}
 	}
```
Comment (Maxmystere) [inline ultrasonic/src/main/java/org/moire/ultrasonic/util/Util.java:175]: Move all function to one Util function

File: ultrasonic/src/main/res/values/themes.xml [hunk 2]
```diff
@@ -1,5 +1,63 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
+    <style name="UltrasonicTheme.Black" parent="Theme.MaterialComponents">
```
Comment (Maxmystere) [inline ultrasonic/src/main/res/values/themes.xml:3]: Using material design component (Amoled)

**Comment by Maxmystere** [comment]
I don't know how translation should be handled so i put default english for strings that I can't translate

**Comment by nitehu** [comment]
That's exactly how translations should be handled ;)
Thanks for your work! We'll check this and get back to you.

**Review by ogarcia — CHANGES_REQUESTED** [review]
This is not more necessary. We remove the translations check because it has not sense and is better to transifex translators. Please remove [this commit](https://github.com/ultrasonic/ultrasonic/pull/326/commits/cd4b1239fca83deb15ab55aa7c2441d2fc1560b4) (or simply remove untranslated strings).

And sorry for the inconvenience :pray:

**Review by nitehu — APPROVED** [review]

**Comment by nitehu** [comment]
I think this will be my default theme 😊

File: ultrasonic/src/main/java/org/moire/ultrasonic/util/Util.java [hunk 3]
```diff
@@ -153,7 +153,25 @@ public static boolean isLockScreenEnabled(Context context)
...
 		SharedPreferences preferences = getPreferences(context);
-		return preferences.getString(Constants.PREFERENCES_KEY_THEME, "dark");
+		return preferences.getString(Constants.PREFERENCES_KEY_THEME, Constants.PREFERENCES_KEY_THEME_DARK);
+	}
+
+	public static void applyTheme(Context context)
+	{
+		String theme = Util.getTheme(context);
+
+		if (Constants.PREFERENCES_KEY_THEME_DARK.equalsIgnoreCase(theme) || "fullscreen".equalsIgnoreCase(theme))
```
Comment (Maxmystere) [inline ultrasonic/src/main/java/org/moire/ultrasonic/util/Util.java:163]: Btw I do not know what this "fullscreen" check is inside that function

Comment (ogarcia) [reply ultrasonic/src/main/java/org/moire/ultrasonic/util/Util.java:163]: Seems legacy code. There is a lot of legacy code in this app... :shrug:

**Review by ogarcia — APPROVED** [review]

**Pull request merged**